src/ansys/fluent/core/launcher/slurm_launcher.py

```diff
@@ -185,43 +205,164 @@ class SlurmLauncher:
+        product_version : FluentVersion, optional
+            Version of Ansys Fluent to launch. Use ``FluentVersion.v241`` for 2024 R1.
+            The default is ``None``, in which case the newest installed version is used.
+        version : str, optional
+            Geometric dimensionality of the Fluent simulation. The default is ``None``,
+            in which case ``"3d"`` is used. Options are ``"3d"`` and ``"2d"``.
+        precision : str, optional
+            Floating point precision. The default is ``None``, in which case ``"double"``
+            is used. Options are ``"double"`` and ``"single"``.
+        processor_count : int, optional
+            Number of processors. The default is ``None``, in which case ``1``
+            processor is used.  In job scheduler environments the total number of
+            allocated cores is clamped to value of ``processor_count``.
+        journal_file_names : str or list of str, optional
+            The string path to a Fluent journal file, or a list of such paths. Fluent will execute the
+            journal(s). The default is ``None``.
+        start_timeout : int, optional
+            Maximum allowable time in seconds for connecting to the Fluent
+            server. The default is ``60``.
+        additional_arguments : str, optional
+            Additional arguments to send to Fluent as a string in the same
+            format they are normally passed to Fluent on the command line.
+        env : dict[str, str], optional
+            Mapping to modify environment variables in Fluent. The default
+            is ``None``.
+        cleanup_on_exit : bool, optional
+            Whether to shut down the connected Fluent session when PyFluent is
+            exited, or the ``exit()`` method is called on the session instance,
+            or if the session instance becomes unreferenced. The default is ``True``.
+        start_transcript : bool, optional
+            Whether to start streaming the Fluent transcript in the client. The
+            default is ``True``. You can stop and start the streaming of the
+            Fluent transcript subsequently via the method calls, ``transcript.start()``
+            and ``transcript.stop()`` on the session object.
+        case_file_name : str, optional
+            Name of the case file to read into the
+            Fluent session. The default is ``None``.
+        case_data_file_name : str, optional
+            Name of the case data file. If names of both a case file and case data file are provided, they are read into the Fluent session.
+        lightweight_mode : bool, optional
+            Whether to run in lightweight mode. In lightweight mode, the lightweight settings are read into the
+            current Fluent solver session. The mesh is read into a background Fluent solver session which will
+            replace the current Fluent solver session once the mesh read is complete and the lightweight settings
```

Comment:
```suggestion
            current Fluent solver session. The mesh is read into a background Fluent solver session, 
            which replaces the current Fluent solver session once the mesh is read and the lightweight settings
```
Style guide says not to use "via" and to use present tense.

Reply:
Done. Thank you.